src/react_native/reanimated.cljs

```diff
@@ -137,3 +139,18 @@
                                  (js-obj "mass"      mass
                                          "damping"   damping
                                          "stiffness" stiffness))))
+
+(defn animate-shared-value-with-decay
+  [anim velocity clamp]
+  (set-shared-value anim
+                    (with-decay (js-obj "velocity" velocity
+                                        "clamp"    (clj->js clamp)))))
```

Comment:
This is more complicated than it needs to be. Call `clj->js` once and remove `js-obj`.

```clojure
(with-decay (clj->js {:velocity velocity :clamp clamp}))
```